src/user.clj

```diff
@@ -0,0 +1,29 @@
+(ns user
```

Comment:
I added this namespace so I could start with `lein repl` and then just call `(start)` from there so I could eval forms in the cljs repl.